spec/models/address_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'rails_helper'
+
+describe 'Address model' do
+  before do
+    @address = Address.new(short_address: 'shortAddr1',
```

Comment:
use factory for this
you need to test the relations also
structure should be
describe relations ....
  describe 
  describe
describe validation ...
  describe
  describe